### PR TITLE
Fix ExtraTorrents provider (use RSS)

### PR DESCRIPTION
**Pull request by medariox**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by fernandog** [comment]
@medariox about pubdate for ET, its already in the other PR.  we need to discuss the better way to store it. so pubdate is on hold for now

File: medusa/providers/torrent/html/extratorrent.py [hunk 1]
```diff
@@ -118,44 +112,22 @@ def parse(self, data, mode):
...
-            # Skip column headers
-            for result in torrent_rows[2:]:
+            for element in elements:
                 try:
-                    cells = result('td')
-
-                    torrent_info = cells[0].find('a')
-                    if not torrent_info:
+                    title = element.title.get_text()
+                    download_url = element.enclosure.get('url')
```
Comment (duramato) [inline medusa/providers/torrent/html/extratorrent.py]: Here you should append our list of trackers like it's done on other trackers

Comment (medariox) [inline medusa/providers/torrent/html/extratorrent.py]: It's not a magnet.

Comment (duramato) [inline medusa/providers/torrent/html/extratorrent.py]: You should use magnets always they are more reliable ,only black hole should use torrent files

**Comment by medariox** [comment]
All done, awaiting approval.

**Comment by medariox** [comment]
This won't be needed for now, as ET removed the html protection they had in place.

**Comment by p0psicles** [comment]
What are we going to do with this now?

**Comment by medariox** [comment]
Let's close it and keep the branch?

**Pull request closed without merging**